doc/lua_api.txt

```diff
@@ -75,6 +75,10 @@ The game directory can contain the following files:
       When both `allowed_mapgens` and `disallowed_mapgens` are
       specified, `allowed_mapgens` is applied before
       `disallowed_mapgens`.
+    * `disallowed_mapgen_flags = <comma-separated mapgen flags>`
```

Comment:
this is extremely misnamed: `mgv5_spflags` is not a mapgen flag, `biomes` would be one
I suggest `disallowed_mapgen_settings`